### DISCUSSION TITLE
Right-click context menu usage

### DISCUSSION THREAD

**Discussion by nlosacco** [bokeh/bokeh#13367] [General]
I understand there is the possibility to endow various elements in a bokeh app with a context menu activated by right-clicking on an element, which, I suppose, can be used to trigger some specific actions. This seems very interesting, although studying the docs I couldn’t understand how to make it work nor there is any example of its usage. Can anyone give me a hint?

**Comment by mattpap** [comment]
Currently there is no support for context menus. It was supposed to be added in bokeh 3.0, but only resulted in `LayoutDOM.context_menu` property that currently does nothing. We need context menu support for our related software stack (e.g. Holoviews), so it should be added hopefully in the near future.

**Comment by levsa** [comment]
What is the status for context menus in Bokeh 3.4.1? We've managed to use a context menu with the existing callbacks like `OpenDialog` and `CustomJS`, but is it possible to add a python callback? What we want to do is to add a menu option to add a point from a plot to the selection (same as a single click does), among other menu alternatives.
